js/main.js

```diff
@@ -0,0 +1,48 @@
+const addBooks = document.querySelector('#add');
+const bookName = document.getElementById('title');
+const authorName = document.getElementById('author');
+
+let bookList = [];
+
+function createBook(bookElement) {
+  document.querySelector('#book-list').innerHTML = '';
+  bookElement.forEach((e) => {
+    const element = document.createElement('books');
```

Comment:
- [ ] There is no element called `books`, even though HTML will create it for you, it is not semantically right.
```suggestion
    const element = document.createElement('li');
```

Reply:
This issue is well resolved! ✔️